Gopkg.lock

```diff
@@ -137,7 +137,7 @@
 
 [[projects]]
   branch = "master"
```

Comment:
I wonder if there's a release tag/branch that we could use instead (probably would be specified in the Gopkg.toml)